dev/com.ibm.ws.microprofile.config.1.3_fat_tck/fat/src/com/ibm/ws/microprofile/config13/tck/Config13TCKLauncher.java

```diff
@@ -41,7 +42,7 @@ public class Config13TCKLauncher {
     public static LibertyServer server;
 
     @ClassRule //EE7 + EE8 repeats
-    public static RepeatTests r = MicroProfileActions.repeat("Config13TCKServer", MicroProfileActions.MP14, MicroProfileActions.MP32);
+    public static RepeatTests r = MicroProfileActions.repeatIf("Config13TCKServer", TCKUtilities::areAllFeaturesPresent, MicroProfileActions.MP14, MicroProfileActions.MP32);
```

Comment:
I don't think it's necessary on this test? Are you just adding it here to test that it doesn't break anything?

Reply:
Nah, dyslexic moment. I read 31 as 13 in a from a list of tcks with this failure.